src/mds/config.py

```diff
@@ -64,7 +64,8 @@ def __init__(self, value):
 
 ADMIN_LOGINS = config("ADMIN_LOGINS", cast=CommaSeparatedLogins, default=[])
 FORCE_ISSUER = config("FORCE_ISSUER", default=None)
-ALLOWED_ISSUERS = set(config("ALLOWED_ISSUERS", cast=CommaSeparatedStrings, default=""))
+ALLOWED_ISSUERS = list(config("ALLOWED_ISSUERS", cast=CommaSeparatedStrings, default=""))
```

Comment:
why list instead of set? I think set will still function as you need

Reply:
No, one of the line in fastapi.py was failing with set. I don't remember which one but I can dig again if you want